docker/docker-compose.yml

```diff
@@ -325,7 +325,7 @@ services:
 
   analytics:
     container_name: supabase-analytics
-    image: supabase/logflare:1.3.28
+    image: supabase/logflare:1.4.1
```

Comment:
this will probably conflict as there was a change to move to 1.4.0 but minor

Reply:
reverted the change